fe/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -126,6 +129,7 @@ public String toString() {
     }
 
     private long dbId;
+    private List<Long> tableIdList;
```

Comment:
How about using a HashSet to speed up the find?

Reply:
actually, in most case ,the table list size is 1, and use list instead of set ,we can reduce memory usage if there are many trasactions with little performance loss